tds-itemselection-common/src/main/java/tds/itemselection/api/IItemSelection.java

```diff
@@ -27,6 +30,8 @@
    * @return ItemGroup
    */
   public ItemGroup getNextItemGroup (SQLConnection connection, ItemCandidatesData itemData)  throws ItemSelectionException;
+
+  ItemGroup getNextItemGroup (SQLConnection connection, ItemCandidatesData itemData, List<ItemGroup> itemGroups)  throws ItemSelectionException;
```

Comment:
I think we should add JavaDocs to all new code that Fairway writes even if previous people did not.